Simulator/Edges/AllEdges.h

```diff
@@ -206,8 +207,9 @@ class AllEdges {
    ///   The weight (scaling factor, strength, maximal amplitude) of the edge.
    vector<BGFLOAT> W_;
 
-   ///  This edge's summation point's address.
-   vector<BGFLOAT *> summationPoint_;
+   ///  This edge's summation point's index.
+   // This was changed from storing the address to index to help serialization
+   vector<int> summationPoint_;
```

Comment:
Just realized that changing `summationPoint_` from storing addresses to indexes makes it a duplicate of the `destVertexIndex_` vector.